hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileSystem.java

```diff
@@ -798,10 +799,11 @@ public void testTrash() throws Exception {
     String testKeyName = "testKey2";
     Path path = new Path(OZONE_URI_DELIMITER, testKeyName);
     ContractTestUtils.touch(fs, path);
-    Assert.assertTrue(trash.getConf().getClass(
-        "fs.trash.classname", TrashPolicy.class).
+    Assert.assertTrue(trash.getConf().getClass("fs.trash.classname",
```

Comment:
this line is an unrelated indentation related change ?

Reply:
Yes, addressed comment.Thanks